src/ui/pages/History/AccelerateTransactionDialog/shared/accelerate-helpers.ts

```diff
@@ -74,10 +74,6 @@ export function increaseGasPrices(
       draft.info.eip1559.fast.max_fee = increase(max_fee, 1.1);
       draft.info.eip1559.fast.priority_fee = increase(priority_fee, 1.3);
     }
-    if (draft.info.optimistic?.l2) {
```

Comment:
Looks like we will not show the difference in speed up for optimistic chains because of this. And this logic should be refactored for them

Reply:
I'm not sure I understand

From the updated calculation code, it looks as if either classic gasPrice or eip1559 is required for the optimistic fee info. Is this correct? @sobolev-igor 
If so, then the increased fee should be displayed. What's the problem with removing this condition?

Reply:
But as I understand it, the displaying algorithm will try to show optimistic fees here, which are constant. The good thing is that applying configuration will work as well

Reply:
> either classic gasPrice or eip1559 is required for the optimistic fee info

correct! That's how speeding up will be reflected in new estimation even for optimistic fees